core/templates/dev/head/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.directive.html

```diff
@@ -19,7 +19,12 @@ <h3>Review Suggestion</h3>
         </angular-html-bind>
       </div>
       <div ng-show="suggestionEditorIsShown">
-        <ck-editor-4-rte ng-model="suggestionData.newSuggestionHtml"></ck-editor-4-rte>
+        <ck-editor-5-rte ng-if="!$ctrl.isDisabled() && !$ctrl.isCurrentRTECKEditor_4()"
```

Comment:
Are you sure isDisabled() is a function in this directive, and are you sure it is needed here?

Reply:
Removed it was not needed + we have ng-show doing the same work.
this and the below one seems to be an independent editor I'll try replacing it will with the schema-editor we use at various places